fe/fe-core/src/main/java/org/apache/doris/nereids/parser/trino/TrinoParser.java

```diff
@@ -0,0 +1,35 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.parser.trino;
+
+import io.trino.sql.parser.ParsingOptions;
+import io.trino.sql.parser.ParsingOptions.DecimalLiteralTreatment;
+import io.trino.sql.parser.SqlParser;
+import io.trino.sql.tree.Statement;
+
+/**
+ * Trino Parser, depends on 395 trino-parser, and 4.9.3 antlr-runtime
```

Comment:
add comment to fe-core pom.xml's antlr dependency. We should not change antlr version since trino parser must use 4.9.3 antlr-runtime

Reply:
OK,  got your point.

Reply:
The version of antlr-runtime in Trino parser is need to be consistant with doris. So the comment is "The version of antlr-runtime in trino parser is need to be consistant with doris, when upgrade antlr-runtime version, should take care of trino-parser"